cmd/ignite/run/exec.go

```diff
@@ -30,5 +31,8 @@ func (ef *ExecFlags) NewExecOptions(vmMatch string, command ...string) (eo *Exec
 
 // Exec executes command in a VM based on the provided ExecOptions.
 func Exec(eo *ExecOptions) error {
+	if err := waitForSSH(eo.vm, constants.SSH_DEFAULT_TIMEOUT_SECONDS, 5); err != nil {
```

Comment:
This may not be a concern, but I noticed that `runSSH()` below accepts a timeout parameter. If this `waitForSSH()` with default timeout of 5 seconds runs before `runSSH()`, the user provided timeout will not work, `waitForSSH()` will return error after waiting for 5 seconds and `runSSH()` with user provided timeout will never be executed.
Should the timeout for `waitForSSH()` be the same as provided in the `ExecOptions.Timeout`?

Reply:
oh yeah, that's a good catch. I've added it there to workaround the behavior introduced by the `--wait` flag and I forgot to remove it. Will fix this up.

Reply:
I think my above conclusion was wrong. Complete removal of `waitForSSH` leads to multinet tests failure when the prometheus socket is up -> VM is started but SSH is still not up yet. So we need it there, but I've changed the 5 second timeout to the user-provided timeout value.